docs/includes/security/jep-290.adoc

```diff
@@ -0,0 +1,92 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+ifndef::rootdir[:rootdir: {docdir}/../..]
+:description: Helidon Security Java Serialization (JEP-290)
+:keywords: helidon, security, serialization, java, JEP-290
+:metainf: META-INF/helidon/serial-config.properties
+
+= Deserialization setup
+
+link:https://openjdk.org/jeps/290[JEP-290] brought support for deserialization filters to Java programming language.
+Such filtering allows us to control which classes may be deserialized using Java serialization.
+
+Helidon default settings forbids any deserialization except for patterns defined in a `pattern`
+property of any `{metainf}` on classpath. The patterns are semicolon delimited strings, such as `io.myapp.&#42;&#42;;java.util.HashMap` (any subpackage of `io.myapp` and class `java.util.HashMap`).
+Helidon will *always* add a deny-all filter pattern to the end of the pattern string (to make sure we exclude any unspecified class - we only operate on whitelists)
```

Comment:
Note: Helidon will *always* add a deny-all filter pattern to the end of the pattern string (to make sure we exclude any unspecified class - we only operate on whitelists).